docs/components/concepts/backups.md

```diff
@@ -0,0 +1,44 @@
+---
+id: backups
+title: "Backups"
+description: "Learn more about Backups in Camunda Platform 8 SaaS."
+---
+
+You can use the backup feature of Camunda Platform 8 SaaS to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) with _zero downtime_. In case of failures that lead to data loss, you can request to restore the backup.
+
+A backup of Camunda Platform 8 SaaS consists of a backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, the backup of a Camunda Platform 8 Cluster is taken as a whole.
+
+With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes.
```

Comment:
Backups are designed specifically for disaster recovery purposes. Backups should not be used for archival of process data.

Reply:
@Sijoma maybe like this is already sufficient for now

Reply:
Thanks for adding that. I think thats the clarity I needed for that. 👍  

Also I'm happy for more review on the content. A LGTM too :D.